src/components/ImageView/index.tsx

```diff
@@ -1,41 +1,81 @@
+import purify from 'dompurify';
 import log from 'loglevel';
 import * as React from 'react';
 
+import { gettext } from '../../utils';
 import styles from './styles.module.scss';
 
-type PublicProps = {
+export type PublicProps = {
```

Comment:
These types are the same:

```diff
diff --git a/src/components/ImageView/index.tsx b/src/components/ImageView/index.tsx
index 26934ef..946a3e0 100644
--- a/src/components/ImageView/index.tsx
+++ b/src/components/ImageView/index.tsx
@@ -5,14 +5,16 @@ import * as React from 'react';
 import { gettext } from '../../utils';
 import styles from './styles.module.scss';
 
-export type PublicProps = {
+type ConvertImageDataParams = {
   _btoa?: typeof btoa;
   _log?: typeof log;
   _sanitize?: typeof purify.sanitize;
-  mimeType: string;
   content: string;
+  mimeType: string;
 };
 
+export type PublicProps = ConvertImageDataParams;
+
 // From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Firefox.
 const supportedImageMimeTypes = [
   'image/jpeg',
@@ -26,14 +28,6 @@ const supportedImageMimeTypes = [
   'image/webp',
 ];
 
-type ConvertImageDataParams = {
-  _btoa?: typeof btoa;
-  _log?: typeof log;
-  _sanitize?: typeof purify.sanitize;
-  content: string;
-  mimeType: string;
-};
-
 const convertImageData = ({
   _btoa = btoa,
   _log = log,
```

I don't think you even need an alias. I was just using it to illustrate the issue.